src/routes/controllers/users.controller.js

```diff
@@ -117,3 +117,27 @@ export async function getReadingStats(req, res) {
     return errorResponse(res, 500, error.message);
   }
 }
+
+/**
+ *
+ * @param {Object} req express request
+ * @param {Object} res express response
+ * @returns {Array} users comments
+ */
+export async function getUsersComments(req, res) {
+  const { user } = req;
+
+  try {
+    const userComments = await Comment.findAll({
```

Comment:
We can include article title in the response for each comment in the response.

Reply:
This has been resolved, see picture below.
<img width="1440" alt="Screenshot 2019-04-24 at 12 14 38 PM" src="https://user-images.githubusercontent.com/31692932/56655956-1a0c9e80-668c-11e9-8173-b16e73719818.png">